<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(0,0,0);
}

QPushButton{
background-color:#000000ff;
border-radius:20px;
}

QPushButton:hover{
background-color: rgb(53,53,79);
border-radius: 20px;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font:87 12pt&quot;Arial Black&quot;;
color: rgb(20,200,220);</string>
            </property>
            <property name="text">
             <string>Trabajo de algo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>499</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimizar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/signo-menos.png</normaloff>../../../Downloads/signo-menos.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_restaurar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/minimizar (1).png</normaloff>../../../Downloads/minimizar (1).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximizar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/maximizar.png</normaloff>../../../Downloads/maximizar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cerrar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/eliminar.png</normaloff>../../../Downloads/eliminar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="menu_lateral">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(53,53,79);
}

QPushButton{
font: 87 12pt&quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20,200,220);
border-radius:5px;
border:1px solid white;
}

QPushButton:hover{
background-color:black;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="bt_1">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>AGREGAR</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../../Downloads/agregar.png</normaloff>../../../Downloads/agregar.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_2">
               <property name="text">
                <string>ABRIR</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../../Downloads/carpeta.png</normaloff>../../../Downloads/carpeta.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_3">
               <property name="text">
                <string>EDITAR</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../../Downloads/editar.png</normaloff>../../../Downloads/editar.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bt_4">
               <property name="text">
                <string>jejeej</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../../Downloads/calendario.png</normaloff>../../../Downloads/calendario.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="styleSheet">
                <string notr="true">QFrame{
background-color: rgb(53,53,79);
}

QPushButton{
font: 87 12pt&quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20,200,220);
border-radius:5px;
border:1px solid #14C8DC;
}

QPushButton:hover{
background-color:black;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="bt_menu1">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../../Downloads/flecha-izquierda.png</normaloff>../../../Downloads/flecha-izquierda.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_menu2">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../../Downloads/flecha-correcta.png</normaloff>../../../Downloads/flecha-correcta.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="styleSheet">
                   <string notr="true">QLineEdit{
font: 75 12pt&quot;Times New Roman&quot;;
background-color:rgb(0,0,0);
color: rgb(255,255,255);
border-radius:8px;
border:1px solid #14c8dc;
}
</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_11">
                  <property name="styleSheet">
                   <string notr="true">QFrame{
background-color: rgb(53,53,79);
}

QPushButton{
font: 87 12pt&quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20,200,220);
border-radius:5px;
border:0px solid #14C8DC;
}

QPushButton:hover{
background-color:black;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../../Downloads/buscar.png</normaloff>../../../Downloads/buscar.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="styleSheet">
                <string notr="true">QFrame{
background-color: rgb(53,53,79);
}

QPushButton{
font: 87 12pt&quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20,200,220);
border-radius:5px;
border:1px solid #14C8DC;
}

QPushButton:hover{
background-color:black;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QToolBox" name="toolBox">
                  <property name="styleSheet">
                   <string notr="true">QToolBox::tab{
background-color:rgb(255,255,255);
border-radius:5px;
color:rgb(0,0,0);
font: 75 12pt &quot;MS Shell Dig 2&quot;;
}

QToolBox::tab:selected{
background-color:rgb(20,200,220);
color:rgb(0,0,0);
font: 75 12pt &quot;MS Shell Dig 2&quot;;
}</string>
                  </property>
                  <property name="currentIndex">
                   <number>2</number>
                  </property>
                  <widget class="QWidget" name="page">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>318</width>
                     <height>394</height>
                    </rect>
                   </property>
                   <attribute name="icon">
                    <iconset>
                     <normaloff>add-icon.png</normaloff>add-icon.png</iconset>
                   </attribute>
                   <attribute name="label">
                    <string>Pagina 1</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="text">
                       <string>No se q onda</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_2">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>318</width>
                     <height>394</height>
                    </rect>
                   </property>
                   <attribute name="label">
                    <string>Pagina 2</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="text">
                       <string>holiiiiiiiiiiiiii</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="page_11">
                   <attribute name="label">
                    <string>Pagina 3</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QPushButton" name="pushButton_3">
                      <property name="text">
                       <string>un beso!!!</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_paginas">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(53,53,79);
}

QLabel{
font:87 12pt &quot;Arial Black&quot;;
background-color: #000000ff;
color: rgb(20,20,220);
border:0px solid #14C8DC;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <widget class="QWidget" name="pagina1">
                <widget class="QLabel" name="label_2">
                 <property name="geometry">
                  <rect>
                   <x>6</x>
                   <y>12</y>
                   <width>301</width>
                   <height>511</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../../../Downloads/agregar.png</pixmap>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="pagina2">
                <widget class="QLabel" name="label_3">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>10</y>
                   <width>311</width>
                   <height>521</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 6pt &quot;MS Shell Dlg 2&quot;;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../../../Downloads/carpeta.png</pixmap>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="pagina3">
                <widget class="QLabel" name="label_4">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>10</y>
                   <width>301</width>
                   <height>501</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../../../Downloads/editar.png</pixmap>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="pagina4">
                <widget class="QLabel" name="label_5">
                 <property name="geometry">
                  <rect>
                   <x>16</x>
                   <y>10</y>
                   <width>301</width>
                   <height>511</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>../../../Downloads/calendario.png</pixmap>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
